plugins/woocommerce/src/Internal/Admin/WcPayWelcomePage.php

```diff
@@ -402,13 +437,13 @@ private function get_incentive(): ?array {
 		// it means there was an API error previously and we should not retry just yet.
 		if ( is_wp_error( $cache ) ) {
 			// Initialize the in-memory cache and return it.
-			$this->incentive = [];
+			$this->incentive = array();
```

Comment:
Just for my knowledge - is this more efficient than using an array literal?

Reply:
It is a requirement from the WC linter to use long array syntax, not the short one. I believe it is to be in alignment with WP. It doesn't affect performance either way.